src/decorators/debounce.ts

```diff
@@ -1,6 +1,12 @@
-import lodashDebounce, {DebounceSettings} from 'lodash/debounce';
+import debounce from 'lodash-es/debounce';
 
-export default function debounce(wait = 500, options?: DebounceSettings) {
+export interface DebounceSettings {
```

Comment:
That import should be stripped out when TS processes the file. You can check by inspecting the built output, though.